server/xml.server.php

```diff
@@ -91,6 +91,7 @@
     // If the method is the same as the action being called
     // Then let's call this function!
     if ($_GET['action'] == $method) {
+        $_GET['format'] = 'xml';
```

Comment:
Do we want to force the format or should we set `$_GET['format']` only if not set by user?